pylint/config/_pylint_config/utils.py

```diff
@@ -0,0 +1,33 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Utils for the 'pylint-config' command."""
+
+
+import sys
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+
+SUPPORTED_FORMATS = {"t", "toml", "i", "ini"}
+
+
+def get_and_validate_format() -> Literal["toml", "ini"]:
+    """Make sure that the output format is either .toml or .ini."""
+    # pylint: disable-next=bad-builtin
+    format_type = input(
+        "Please choose the format of configuration, (T)oml or (I)ni (.cfg): "
+    ).lower()
+
+    if format_type not in SUPPORTED_FORMATS:
+        raise ValueError(
+            f"Format should be one of {', '.join(i.capitalize() for i in sorted(SUPPORTED_FORMATS))}"
+        )
```

Comment:
It doesn't really take a lot of time to restart the process. So for now I think this should be fine.

It would be a nice enhancement though, but perhaps we should then do something with a decorator `@fatfinger_user_input` on all validation functions? Something to think about I guess.

Reply:
The `--toml` thing is indeed something I would like to add.

I'm thinking of `pylint-config generate --interactive --format=toml` and have this validation function check `linter.config.format` before calling `input()`.

Reply:
> @fatfinger.user_input

The name is currently free on pypi btw ! :heavy_wink:

Reply:
For the retrying, see https://github.com/PyCQA/pylint/pull/6653.

Just a little hack to retry and allow users to exit gracefully.